docs/databags.md

```diff
@@ -109,6 +109,7 @@ An apps hash is a contextual (see above) Hash, part of a plaintext data bag item
 * `[app]['download']['version']` - Expected [version number][app.conf] (if any) used to determine if a new app should be downloaded.
 * `[app]['files']` - Hash of files to manage under the "default" or "local" directory.
 * `[app]['files'][filename]` - Contents of a particular file to manage. It can take 3 values, a hash of stanzas -> key-value pairs (then written with the splunk template), a string (written as is), or nil / false (deleted). If the hash or string is empty, the file is also deleted.
+* `[app]['lookups']` - Hash of lookup files for the app. The key in the hash will be the name of file when it lands on splunk and value will be the url to lookup file. To delete an existing lookup file, set the value of the lookup file to empty or to `"nil"`. Only supported file name formats are .csv, .gz and .kmz. If you are downloading a comma separated text file (.txt), the file name you specify as the key in the hash should have a .csv extension.  
```

Comment:
Saw this in the lookup upload view in the splunk web UI
>Select either a plaintext CSV file, a gzipped CSV file, or a KMZ file. 
The maximum file size that can be uploaded through the browser is 500MB.
Enter the name this lookup table file will have on the Splunk server. If you are uploading a gzipped CSV file, enter a filename ending in ".gz". If you are uploading a plaintext CSV file, we recommend a filename ending in ".csv". For a KMZ file, we recommend a filename ending in ".kmz".

Thought of giving the value always the same type. Agreed that it was not a good idea. Updated the option to be null if the lookup needs to be deleted.  8a626089e661f787e149847348f693986cd49e45